src/util_test.cc

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <algorithm>
```

Comment:
Should be after `util.h`, see https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes